recipes/cairo/all/conanfile.py

```diff
@@ -0,0 +1,245 @@
+                env_build = self._get_env_build()
+                env_build.install()
+        tools.remove_files_by_mask(self.package_folder, "*.la")
+        
+        self.copy("COPYING*", src=self._source_subfolder, dst="licenses", keep_path=False)
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+
+    def package_info(self):
+        self.cpp_info.components["cairo_"].names["pkg_config"] = "cairo"
+        self.cpp_info.components["cairo_"].libs = ["cairo"]
+        self.cpp_info.components["cairo_"].includedirs.insert(0, os.path.join('include', 'cairo'))
+        self.cpp_info.components["cairo_"].requires = ["pixman::pixman", "libpng::libpng", "zlib::zlib"]
+        if self.options.with_freetype:
+            self.cpp_info.components["cairo_"].requires.append("freetype::freetype")
+        
+        if self.settings.os == "Windows":
+            if not self.options.shared:
+                self.cpp_info.components["cairo_"].defines.append('CAIRO_WIN32_STATIC_BUILD=1')
+        else:
+            if self.options.with_glib:
+                self.cpp_info.components["cairo_"].requires.extend(["glib::gobject-2.0", "glib::glib-2.0"])
+            if self.options.with_fontconfig:
+                self.cpp_info.components["cairo_"].requires.append("fontconfig::fontconfig")
+        if self.settings.os == "Linux":
+            if self.options.with_xcb:
+                self.cpp_info.components["cairo_"].requires.extend(["xorg::xcb-shm", "xorg::xcb"])
+            if self.options.with_xlib_xrender:
+                self.cpp_info.components["cairo_"].requires.extend(["xorg::xcb-render"])
+            if self.options.with_xlib:
+                self.cpp_info.components["cairo_"].requires.extend(["xorg::x11", "xorg::xext"])
+
+
+        if self.settings.os == "Windows":
+            self.cpp_info.components["cairo-win32"].names["pkg_config"] = "cairo-win32"
+            self.cpp_info.components["cairo-win32"].requires = ["cairo_", "pixman::pixman", "libpng::libpng"]
+
+        if self.options.with_glib:
+            self.cpp_info.components["cairo-gobject"].names["pkg_config"] = "cairo-gobject"
+            self.cpp_info.components["cairo-gobject"].libs = ["cairo-gobject"]
+            self.cpp_info.components["cairo-gobject"].requires = ["cairo_", "glib::gobject-2.0", "glib::glib-2.0"]
+        if self.settings.os != "Windows":
+            if self.options.with_fontconfig:
+                self.cpp_info.components["cairo-fc"].names["pkg_config"] = "cairo-fc"
+                self.cpp_info.components["cairo-fc"].requires = ["cairo_", "fontconfig::fontconfig"]
+            if self.options.with_freetype:
+                self.cpp_info.components["cairo-ft"].names["pkg_config"] = "cairo-ft"
+                self.cpp_info.components["cairo-ft"].requires = ["cairo_", "freetype::freetype"]
+            self.cpp_info.components["cairo-pdf"].names["pkg_config"] = "cairo-pdf"
```

Comment:
Neither `cairo-win32`, `cairo-fc` or `cairo-pdf` have libraries.
Is this correct?

Reply:
that's what I saw in the generated upstream .pc files, yes